backend/src/middleware/slugify/uniqueSlug.js

```diff
@@ -13,3 +13,17 @@ export default async function uniqueSlug (string, isUnique) {
   } while (!await isUnique(uniqueSlug))
   return uniqueSlug
 }
+
+export function isUniqueFor (context, type) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-06T23:31:14Z" title="Tuesday, May 7th 2019, 1:31:14 am +02:00">May 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -13,3 +13,17 @@ export default async function uniqueSlug (string, isUnique) {
   } while (!await isUnique(uniqueSlug))
   return uniqueSlug
 }
+
+export function isUniqueFor (context, type) {
```


This is just a refactoring? :thinking: or do you use it in another place, too?